build/documentation/copy-package-readme.js

```diff
@@ -0,0 +1,70 @@
+/**
+ * Utility for copying readme to .docs/en/packages/[package-name]/readme.md.
+ * Usage: node build/documentation/copy-package-readme.js
+ */
+const path = require("path");
+const fs = require("fs");
+const glob = require("glob");
+
+const rootDir = path.resolve(process.cwd());
+const srcReadmePaths = "packages/*/README.md";
+const destDir = "docs\\en\\packages";
```

Comment:
this `destDir` is now causing the script to throw in my environment.

I would recommend using `path.join("docs", "en", "packages")`. Alternatively,  you can construct the path using `path.sep` to easily allow Node to pick the path separator based on runtime environment.

Reply:
Ahhh this feedback was very helpful, tyty

Reply:
Of course! Thanks for making the update :)